pandas/tests/tseries/offsets/test_offsets_properties.py

```diff
@@ -85,8 +89,10 @@
 # Offset-specific behaviour tests
 
 
-# Based on CI runs: Always passes on OSX, fails on Linux, sometimes on Windows
-@pytest.mark.xfail(strict=False, reason="inconsistent between OSs, Pythons")
+@pytest.mark.skipif(
```

Comment:
use td.skip_if_no with min_version

Reply:
Thank you, didn't even know that decorator exist.